common.flag.mk

```diff
@@ -4,38 +4,28 @@ DYNLINK ?= false
 # For COMMON
```

Comment:
We'll want to migrate this to a CMake project so we can tie in the flags used to build dyninst.